### PR TITLE
Docs: Gauge and Bar Gauge updates

### PR DESCRIPTION
**Pull request by Develer**
Related to https://github.com/grafana/grafana/pull/76216 + https://github.com/grafana/grafana/pull/76042

**What is this feature?**

Update Gauge / Bar Gauge docs with new sizing mode and other gauge dimension options.

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/bar-gauge/index.md [hunk 1]
```diff
@@ -103,6 +103,12 @@ Limit the minimum height of the bar row in the horizontal direction.
 
 Automatically show y-axis scrollbar when there is a large amount of data.
 
+### Max height
+
+Limit the maximum height of the bar row in the horizontal direction.
+
+Automatically show y-axis scrollbar when there is a large amount of data.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/bar-gauge/index.md:110]:
```suggestion
Limit the maximum height of the bar row when the gauge is oriented horizontally.

Automatically show y-axis scrollbar when there's a large amount of data.
```

**Review by imatwawana — COMMENTED** [review]
Thanks for updating! I know this is outside the scope of your PR, but can you please make the change I've suggested also on lines 96, 98, 102, and 104 as well? "Direction" isn't really the correct word here, so we should fix it in all three places. The other change is just a small style fix (we use contractions where possible).

File: docs/sources/panels-visualizations/visualizations/bar-gauge/index.md (hunk 1 above)
Comment (Develer) [reply docs/sources/panels-visualizations/visualizations/bar-gauge/index.md:110]: @imatwawana sure, no problem, I'll update it, we didn't merge [this PR](https://github.com/grafana/grafana/pull/76042) yet but as soon as we merge it, I'll update this one 🙂

**Comment by Develer** [comment]
@nmarrs thank you for updating!
LGTM! 🚀

**Review by nmarrs — APPROVED** [review]

**Pull request merged**